README.md

```diff
@@ -2,25 +2,39 @@
+| 1.14          | Foxy - Humble - Rolling | [release/1.14](https://github.com/PX4/px4_msgs/tree/release/1.14) |
+| `main` branch | Foxy - Humble - Rolling | [main](https://github.com/PX4/px4_msgs)                           |
 
-When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository change, a CI/CD pipeline automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.
+### Manual messages synchronization
+
+When PX4 message definitions in the `main` branch of [PX4 Autopilot](https://github.com/PX4/Autopilot) change, a CI/CD pipeline automatically copies and pushes updated ROS message definitions to this repository. This ensures that this repository `main` branch and the PX4-Autopilot `main` branch are always up to date.
+However, if you are using a custom PX4 version and you modified existing messages or created new one, then you have to manually synchronize them in this repository:
+
+- Checkout the correct branch associated to the PX4 version from which you detached you custom version.
+- Delete all `*.msg` files in `msg/` and copy all `*.msg` files from `PX4-Autopilot/msg/` in it. Assuming that this repository and the PX4-Autopilot repository are placed in your home folder, you can run:
+  ```sh
+  rm -f ~/px4_msgs/msg/*.msg
+  cp ~/PX4-Autopilot/msg/*.msg ~/px4_msgs/msg/
+  ```
 
 ## Install, build and usage
 
-Check the [`Ament` tutorial](https://index.ros.org/doc/ros2/Tutorials/Ament-Tutorial/) to understand how this can be built inside a worspace. Check the [RTPS/ROS2 Interface](https://dev.px4.io/en/middleware/micrortps.html) section on the PX4 Devguide for further details on how this integrates with the [`px4_ros_com` package](https://github.com/PX4/px4_ros_com).
+Check [Using colcon to build packages](https://docs.ros.org/en/humble/Tutorials/Beginner-Client-Libraries/Creating-Your-First-ROS2-Package.html#build-a-package) to understand how this can be built inside a workspace. Check the [PX4 ROS 2 User Guide](https://docs.px4.io/main/en/ros/ros2_comm.html) section on the PX4 documentation for further details on how this integrates PX4 and how to exchange messages with the autopilot.
 
 ## Bug tracking and feature requests
 
 Use the [Issues](https://github.com/PX4/px4_msgs/issues) section to create a new issue. Report your issue or feature request [here](https://github.com/PX4/px4_msgs/issues/new).
 
 ## Questions and troubleshooting
 
-Reach the PX4 development team on the `#messaging` PX4 Slack channel:
-[![Slack](https://px4-slack.herokuapp.com/badge.svg)](http://slack.px4.io)
+Reach the PX4 development team on the PX4 Discord Server:
```

Comment:
Can you please move the badge to the top of the readme? I think its best if it matches what we have on PX4 Autopilot repo.

Reply:
Done!